.github/actions/install-happy/action.yaml

```diff
@@ -17,6 +17,10 @@ inputs:
     description: Whether to add happy to the system bin directory
     required: false
     default: ""
+  version_lock_file:
```

Comment:
Could you also add this to https://github.com/chanzuckerberg/github-actions/blob/main/.github/actions/deploy-happy-stack/action.yml and pass it through since that's how the install-happy action is often invoked?

Reply:
Good idea! I'll enter a ticket for it.